src/ontology/ro-edit.owl

```diff
@@ -5646,6 +5648,18 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0004047 "2018-03-13T23:55:19Z"
 AnnotationAssertion(rdfs:label obo:RO_0004047 "causally upstream of or within, positive effect")
 SubObjectPropertyOf(obo:RO_0004047 obo:RO_0002418)
 
+# Object Property: obo:RO_0007000 (has_driver)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0007000 "A relation between a material entity and a system, in which the material entity is any natural or human-influenced factor that directly or indirectly causes a change in the system.")
+AnnotationAssertion(rdfs:label obo:RO_0007000 "has_driver")
```

Comment:
- Label should be without underscore
- You should add your ORCID and  github issue to both relations:
AnnotationAssertion(dc:contributor obo:RO_0007000 <https://orcid.org/0000-0002-6601-2165>)
AnnotationAssertion(rdfs:seeAlso obo:RO_0007000 <https://github.com/oborel/obo-relations/issues/506>)

Reply:
I have edited the file as requested. 
Thank you for the help !! 
Cheers,
Lynn

Reply:
Thanks!